README.md

```diff
@@ -77,12 +77,11 @@ make DEBUG=1  # compile optionally with PRINTF
 
 You can choose which device to compile and load for by setting the `BOLOS_SDK` environment variable to the following values :
 
-* `BOLOS_SDK=$NANOS_SDK`
 * `BOLOS_SDK=$NANOX_SDK`
 * `BOLOS_SDK=$NANOSP_SDK`
 * `BOLOS_SDK=$STAX_SDK`
 
-By default this variable is set to build/load for Nano S.
+By default this variable is set to build/load for Nano S+.
```

Comment:
Do we want this behavior? Nothing => no build ?

Reply:
I can remove this line